fcl/gen/corsika/prodcorsika_icarus_RootInput/prodcorsika_protononly_icarus_ROOTinput.fcl

```diff
@@ -0,0 +1,42 @@
+#
+# File:    prodcorsika_protononly_icarus_ROOTinput.fcl
+# Purpose: Overlay of cosmic rays using only proton primaries (not CMC) onto signal events.
+# Author:  Jamie Dyer (jamie.dyer@colostate.edu)
+# Date:    December 20, 2021
+#
+#This job configuration uses 'prodcorsika_protononly_icarus.fcl' to overlay 
+#cosmic ray events onto signal events provided in RootInput (as opposed to 
+#empty events).
+
+#'prodcorsika_protononly_icarus.fcl' generates cosmic ray events in ICARUS with a 
+#configuration similar to 'prodcorsika_standard_icarus.fcl', but using 
+#a proton-only composition instead of the Contant Mass Composition.
+#
+#
+# 
+# Output
+# -------
+# 
+# See 'prodcorsika_protononly_icarus.fcl' and `prodcorsika_standard_icarus.fcl`.
+# The main output is an _art_ ROOT file with content:
+#  * `generator`: generated cosmic rays from CORSIKA (`simb::MCTruth` collection)
+# 
+
+#include "prodcorsika_protononly_icarus.fcl"
+
+
+process_name: Gen
+
+source:
+{
+	module_type: RootInput
+#	skipEvents: 0
+}
+
```

Comment:
I think the RootInput needs to stay. If I try to input the root file without including RootInput for the source, eg directly with the preexisting prodcorsika fcl file, I get the following error:

```
$ lar -c prodcorsika_protononly_icarus.fcl -s /pnfs/icarus/persistent/users/jdyer/NovPR_ROOTfiles/Smass_150MeV/split_files_icarus_HiggsScalar150MeV_0001_20211202T042014-SplitHPFiles.root 
OptionsHandler caught a cet::exception calling art::BasicPostProcessor::doProcessOptions()
---- Configuration BEGIN
  Error: source files specified for EmptyEvent source.
---- Configuration END
Art has completed and will exit with status 90.
```

Reply:
Got it. It's because the included file already defines a `source`. So my suggestion would translate into _removing_ that (`source: @erase`), but I do not recommend this since it's quite obscure. I consider this thread solved.